scripts/find_pulsar_in_obs.py

```diff
@@ -37,7 +37,8 @@
 #MWA scripts
 import sn_flux_est as sfe
 from mwa_pb import primary_beam
-from mwa_metadb_utils import mwa_alt_az_za, getmeta, get_common_obs_metadata, get_obs_array_phase
+from mwa_metadb_utils import mwa_alt_az_za, getmeta, get_common_obs_metadata,\
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused getmeta imported from mwa_metadb_utils](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4514653)